aspnetcore/migration/70-80.md

```diff
@@ -73,64 +73,541 @@ In the project file, update each [Microsoft.AspNetCore.*](https://www.nuget.org/
 
-    private string BoundValue
-    {
-        get => ChildMessage ?? string.Empty;
-        set => ChildMessageChanged.InvokeAsync(value);
-    }
-}
-```
+1. Add an entry to the `_Imports.razor` file to make shorthand render modes available to the app:
 
-In .NET 7, you can use the new `@bind:get` and `@bind:set` modifiers to support two-way data binding and simplify the binding syntax:
+   ```razor
+   @using static Microsoft.AspNetCore.Components.Web.RenderMode
+   ```
 
-```razor
-<NestedGrandchild @bind-GrandchildMessage:get="ChildMessage" 
-    @bind-GrandchildMessage:set="ChildMessageChanged" />
-```
+1. Move the content in the `_Host` page (`Pages/_Host.cshtml`) to the empty `App.razor` file. Proceed to make the following changes to the `App` component.
 
-> [!IMPORTANT]
-> The `@bind:get` and `@bind:set` features are receiving further updates at this time. To take advantage of the latest updates, confirm that you've installed the [latest SDK](https://dotnet.microsoft.com/download/dotnet/7.0).
->
-> Using an event callback parameter (`[Parameter] public EventCallback<string> ValueChanged { get; set; }`) isn't supported. Instead, pass an <xref:System.Action>-returning or <xref:System.Threading.Tasks.Task>-returning method to `@bind:set`.
->
-> For more information, see the following resources:
->
-> * [Blazor `@bind:after` not working on .NET 7 RTM release (dotnet/aspnetcore #44957)](https://github.com/dotnet/aspnetcore/issues/44957)
-> * [`BindGetSetAfter701` sample app (javiercn/BindGetSetAfter701 GitHub repository)](https://github.com/javiercn/BindGetSetAfter701)
+   > [!NOTE]
+   > In the following example, the project's namespace is `Net8BlazorServer`. Adjust the namespace to match your project.
+
+   Remove the following lines from the top of the file:
+
+   ```diff
+   - @page "/"
+   - @using Microsoft.AspNetCore.Components.Web
+   - @namespace Net8BlazorServer.Pages
+   - @addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+   ```
+   
+   Replace the preceding lines with the following:
+
+   ```razor
+   @inject IHostEnvironment Env
+   ```
+
+   Remove the tilde (`~`) from the `href` of the `<base>` tag:
+
+   ```diff
+   - <base href="~/" />
+   + <base href="/" />
+   ```
+
+   Remove the Component Tag Helper for the `HeadOutlet` component:
+
+   ```diff
+   - <component type="typeof(HeadOutlet)" render-mode="ServerPrerendered" />
+   ```
+
+   Replace the preceding line's tag helper with the `HeadOutlet` component:
+
+   ```razor
+   <HeadOutlet @rendermode="RenderMode.InteractiveServer" />
+   ```
+
+   Remove the Component Tag Helper for the `App` component:
+
+   ```diff
+   - <component type="typeof(App)" render-mode="ServerPrerendered" />
+   ```
+
+   Replace the preceding line's tag helper with the `Routes` component:
+
+   ```razor
+   <Routes @rendermode="RenderMode.InteractiveServer" />
+   ```
```

Comment:
````suggestion
   Replace the Component Tag Helper for the `App` component with the `App` component:

   ```diff
   - <component type="typeof(App)" render-mode="ServerPrerendered" />
   + <Routes @rendermode="InteractiveServer" />
   ```
````

Reply:
Ditto ... and btw, this is a standard Blazor docs convention to make the **Copy** button work well. I'd have had the doc engineers make the **Copy** button smarter, but it really just copies everything, and the dev has to back out the deletion lines and the "+ " from the fronts of the addition lines. I recommend we go with my approach, but I'll change it here (and everywhere across the Blazor docs) if you wish.

Reply:
Ah, I should've known there was a good reason you structured things this way :smile:. My concern in this case is this pattern makes the doc much longer. Did we get customer complaints about the copy button not working previously when using the diff syntax?

Reply:
***I complain!*** 😆 ... and I'd be happy to open an issue for them or ask the team to forward it, but they ignore me constantly.

I agree on the length, but it's a PITA to use DIFFs the way they're proposing ... they way they do. Still tho ... if you want me to change them all, no problem, of course. However, it's a standard Blazor docs convention, so it's done this way *everywhere* to save our readers a lot of time and frustration. It will need to be changed in many docs. I'll do that tho on a follow-up issue tho. We need to get out of this one soon to free it up for the other doc authors.

Reply:
😆 Fair enough. Maybe we should just stick with the existing convention then.